content/news/2016/05/stsi-to-sponsor-devopsdaysdc.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
I'm not sure it helps us to recreate 4 year old news, unless we're getting lots of hits at that URL.

Reply:
We aren't getting lots of hits there at all. Reducing 404s was the only reason if it is deemed a bad "look", sure not a priority in my book.